v-next/hardhat-keystore/src/internal/loaders/keystore-file-loader.ts

```diff
@@ -21,6 +27,10 @@ export class KeystoreFileLoader implements KeystoreLoader {
     return this.#keystoreFilePath;
   }
 
+  public getKeystoreDevPasswordFilePath(): string {
+    return this.#keystoreDevPasswordFilePath;
+  }
```

Comment:
Nit: I think I would go with a public property rather than a getter.
It isn't adding anything, so it prompts the questions from a consumer, oh they haven't gone for a public property here there must be something more going on.